### PR TITLE
Add mastodon link

### PR DESCRIPTION
**Pull request by jakebailey**
For https://github.com/microsoft/TypeScript/issues/56396

### CODE REVIEW

File: packages/typescriptlang-org/src/components/layout/SiteFooter.tsx [hunk 1]
```diff
@@ -116,6 +116,12 @@ const communityLinks = [
   {
     title: "@TypeScript",
     url: "https://twitter.com/TypeScript",
+    me: true,
+  },
+  {
+    title: "@TypeScript",
```
Comment (jakebailey) [inline packages/typescriptlang-org/src/components/layout/SiteFooter.tsx]:
```suggestion
    title: "Mastodon",
```

File: packages/typescriptlang-org/src/components/layout/SiteFooter.tsx [hunk 2]
```diff
@@ -116,6 +116,12 @@ const communityLinks = [
   {
     title: "@TypeScript",
     url: "https://twitter.com/TypeScript",
+    me: true,
```
Comment (DanielRosenwasser) [inline packages/typescriptlang-org/src/components/layout/SiteFooter.tsx]: is this actually used by Twitter?

Comment (jakebailey) [reply packages/typescriptlang-org/src/components/layout/SiteFooter.tsx]: Nah. I'll remove it.

Comment (jakebailey) [reply packages/typescriptlang-org/src/components/layout/SiteFooter.tsx]:
```suggestion
```

**Review by DanielRosenwasser — APPROVED** [review]

**Pull request merged**